Spanner/tests/System/SpannerTestCase.php

```diff
@@ -111,11 +152,38 @@ private static function getClient()
         return self::$client;
     }
 
-    public static function getDatabaseInstance($dbName)
+    public static function getDatabaseInstance($dbName, $options = [])
+    {
+        $keyFilePath = getenv('GOOGLE_CLOUD_PHP_TESTS_KEY_PATH');
+
+        return self::$client->connect(self::INSTANCE_NAME, $dbName, $options);
+    }
+
+    public static function getDatabaseFromInstance($dbName, $options = [])
```

Comment:
I am confused with this function and its params. Could it be like:

```suggestion
    public static function getDatabaseFromInstance($instance, $dbName, $options = [])
```